examples/default_values_backend_app/README.md

```diff
@@ -0,0 +1,71 @@
+# Contentful Backend Demo APP
+
+A demo App that will prefill certain values when an entry is created.
+
+The purpose of this demo is to show how backend Apps can be setup and used.
+
+
+## Prerequisites
+This demo requires node.js, yarn, and some way of making the running server
+available to contentful. For local development it is easiest to use
+[ngrok](https://ngrok.com/)
+
+## Setup instructions
+
+1. Run
+shell```
+ngrok http 3543
+```
+Take note of the url provided by NGROK. This step is only necessary when running
+the app locally.
+
+2. Update `.env` with
+shell```
+ORG_ID= # Your contentful organisation id
+SPACE_ID= # Your contentful spaece id
+ENVIRONMENT_ID= # Your contentul enviornment name
+APP_LOCATION=the ngrok URL from step 1
+```
+
+### Warning!
+The setup for this demo will make install an app, and create a content type in
+whatever orgnization/space/environment you use. We highly recommend that you do
+not use your production environment for testing this App.
+
+3. Add a valid CMA token to the environment variables, you can obtain this token
+   from the contentful web app
+
+```shell
+export CMA_TOKEN=${a valid CMA token}
+```
+
+4. create key pair
```

Comment:
Yeah I agree. I've moved it into a general setup script, because I don't think this is the place to explain it. But let's definitely add a link to documentation when we have some.